docs/manual/src/docs/asciidoc/_includes/servlet/oauth2/oauth2-resourceserver.adoc

```diff
@@ -1273,13 +1296,13 @@ public class TenantAuthenticationManagerResolver implements AuthenticationManage
 
 	@Override
 	public AuthenticationManager resolve(HttpServletRequest request) {
-		return this.authenticationManagers.computeIfAbsent(toTenant(request), this::fromTenant); <3>
```

Comment:
Why was this removed? It should be linked to the `<3>` in the comments below.

Reply:
I see... The reference was aimed at the two lines. I thought it was just aimed at the second code referenced. My bad!